### PR TITLE
Added documentation & fixed a few indentation issues. fixes #61

### LINKED ISSUE #61: Document the services

### PR DESCRIPTION
**Pull request by Azkle**
Made the pull request for Gijs.

### CODE REVIEW

**Comment by Azkle** [comment]
LGTM

File: app/scripts/controllers/audio/audioController.js [hunk 1]
```diff
@@ -9,35 +12,46 @@
...
-        }
+      var elem = angular.element('<feedback-sam/>');
+      angular.element('#feedback-sam-container').append(elem);
+      $compile(elem)($scope);
+    };
+
+    /**
+     * Function to handle votes.
+     * @param  orientation, if a song is liked or disliked.
+     * @return nothing
```
Comment (Azkle) [inline app/scripts/controllers/audio/audioController.js]: Maybe delete this return, to be consistent.

**Comment by JWGmeligMeyling** [comment]
CAN BE MERGED

**Comment by JWGmeligMeyling** [comment]
More docs more good, will merge.

**Pull request merged**